.changeset/wise-suns-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+"@twilio-paste/input": patch
+---
+
+[Input] updated focus stylings on increment/decrement buttons for number input type on both default and inverse variants
```

Comment:
praise: love the detailed changeset description, that's something we strive for and helps consumers

issue(blocking): we also include "@twilio-paste/core" on all changesets that are part of the core package. for almost all changes, the core change will be the same as the specific package (so in this case, it would be a patch to core), but for new packages (like the addition of the side panel component), we do a minor to core and a major to the new package, to avoid unnecessary breaking changes to core.

Reply:
whoops I was looking at an old commit, resolving this!